1_fetch/src/1_MendotaPull.R

```diff
@@ -0,0 +1,20 @@
+## This script pulls down model evaluation data for Lake Mendota
+
+## Function for pulling down data from 
+## USGS's ScienceBase https://www.sciencebase.gov/
+
+fetch_data <- function(ID, project_output_dir, file_name){
```

Comment:
Can you be more specific about where this function fetches data from, perhaps both in the function name and in the name of the `ID` argument?

Reply:
Also, `project_output_dir` sounds like the single output dir for the whole project, but our convention is to have a separate output directory for each phase of the project. This naming decision is discussable, with no one right answer, but consider alternatives to what you have now.